### ISSUE TITLE
[Bug]: Same chart on Overview and Single Node view has dimensions with different colours

### ISSUE THREAD

**Issue by hugovalente-pm** [netdata/netdata-cloud#457]
### Bug description

The same chart on Overview and Single Node view has dimensions presented with different colour.

### Expected behavior

The same dimensions for the same chart should have the same colours.

### Steps to reproduce

1. Go to System Ram chart on Overview
2. Go to System Ram chart on a Single Node
3. Check that the dimensions colours are not the same

### Screenshots

**Overview**
![image](https://user-images.githubusercontent.com/82235632/172641376-f26d1312-9c4f-43fa-b9d3-eabf283f6a91.png)

**Single Node view**
![image](https://user-images.githubusercontent.com/82235632/172641415-5b942cf2-1be6-44dd-8dd4-8f622016e70a.png)

### Desktop

Browser Version [e.g. 22]

**Comment by MrZammler** [comment]
Hi @hugovalente-pm 

I did some small checks with a recent enough agent version. What I observed, was even though the colors where the same in overview and single node views for the same chart, in overview the dimensions where not in the correct order.

I checked the response from the browser's network tab that the FE gets, and I think I got different order than what the agent responded for that request...

Should we check it a bit more? Sorry I'm not familiar with the change on latest agents regarding the metadata, just whether we need to check a bit...
